Riot/Views/RoomInputToolbar/RoomInputToolbarView.m

```diff
@@ -78,18 +78,22 @@ - (void)awakeFromNib
 -(void)customizeViewRendering
 {
     [super customizeViewRendering];
-    
+    [self userInterfaceThemeDidChange];
+}
+
+- (void)userInterfaceThemeDidChange
+{
     // Remove default toolbar background color
     self.backgroundColor = [UIColor clearColor];
-    
+
     self.separatorView.backgroundColor = kRiotAuxiliaryColor;
-    
+
     // Custom the growingTextView display
     growingTextView.layer.cornerRadius = 0;
     growingTextView.layer.borderWidth = 0;
     growingTextView.backgroundColor = [UIColor clearColor];
-    
-    growingTextView.font = [UIFont systemFontOfSize:15];
+
+    growingTextView.font = kRiotTextFontBody;
```

Comment:
About `RoomInputToolbarView`, I agree to apply this change, but the corresponding xib file must be adjusted. Indeed the icons are not aligned when the user selects a large text size.
See the following screenshot with the view frames:

![screen shot 2017-10-30 at 19 15 09](https://user-images.githubusercontent.com/8969772/32187990-d438104a-bda6-11e7-9603-2407344b1b00.png)

I know this is not your fault ;). We may update `RoomInputToolbarView` class in another PR if you prefer.